ElectricityBillApi/Models/ElectricProvider.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace ElectricityBillApi.Models
+{
+    public class ElectricProvider
+    {
+        private PowerPlant _subscibedPowerPlant;
+
+        public string Name { get; set; }
```

Comment:
Readonly probably is more suitable.

Reply:
I assume you mean the field (like in the next request)...